pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -875,8 +912,10 @@ func doInactivateHalt(ctx *zedmanagerContext,
 	if ns.HasError() {
 		log.Errorf("Received error from zedrouter for %s: %s",
 			uuidStr, ns.Error)
-		status.SetErrorWithSource(ns.Error, types.AppNetworkStatus{},
-			ns.ErrorTime)
+		description := ns.ErrorDescription.Copy()
+		description.ErrorSeverity = types.ErrorSeverityNotice
```

Comment:
How do you know this is Notice? I think zedrouter should set it.
But as the first cut it might make sense to 
1) get the types done
2) set in downloader
3) make the downloader -> blob -> ct -> volume -> app instance/baseos propagation work

You have 1 and 2 already (and some of 3 - too late for me to review).
But the zedrouter and other errors we can leave with their defaults for now and address them in a subsequent error.

Note that the ones which the customer cares about next are:
4. Insufficient memory (in zedmanager)
5. Adapter assigned to other app instance (in domainmgr)

Reply:
4 is also partially done with actually the same question about severely because of no info about retry counts, seems I need to find/add them or use time instead.
Will think about 5, thank you for point me onto it.

Reply:
Those don't have a periodic retry; the retry is triggered when something frees up (memory and adapter, respectively)

Reply:
Sure, but I need calculate severity for them somehow.

Reply:
Just leave at default aka error for now.

Reply:
Actually given the code placement I suggested ways to set it to info vs warning (based on the "halting" check which affects the error strings already.